### PR TITLE
HELM-35: Defer null value handling to opennms.js client

### PR DESCRIPTION
**Pull request by mvrueden**
* JIRA: https://issues.opennms.org/browse/HELM-35

Must be merged after PR https://github.com/OpenNMS/opennms-js/pull/14

### CODE REVIEW

File: src/datasources/fault-ds/mapping/ValueMapping.js [hunk 1]
```diff
@@ -1,34 +1,10 @@
 
 export class ValueMapping {
```
Comment (j-white) [inline src/datasources/fault-ds/mapping/ValueMapping.js]: Is it worth even keeping this class, if all it does is return the given `value`?

Comment (mvrueden) [inline src/datasources/fault-ds/mapping/ValueMapping.js]: Let's keep it for now, until we know that we really don't need it. My plan is to clean up the code when we are ready for v1. I just don't want to delete it now and then re-create it later (-:

**Comment by j-white** [comment]
Can we add a note in the query editor to indicate that 'null' is a valid value for the date fields?

**Comment by j-white** [comment]
Also, should be bump the version of opennms-js that we depend on here? Since https://github.com/OpenNMS/opennms-js/pull/14 has been merged.

**Comment by mvrueden** [comment]
The explanation was added directly in develop with the following commit: https://github.com/OpenNMS/grafana-opennms-helm-app/commit/e4f349eadecc046b52b8c35a5307f419214d8a94

The opennms.js version was bumped directly in develop with the following commit: https://github.com/OpenNMS/grafana-opennms-helm-app/commit/aad57bd8881c992a020bd677f67d09bba39fdbef

**Pull request merged**